test/screens/course_schedule_test.dart

```diff
@@ -5,9 +5,7 @@ import 'package:flutter_test/flutter_test.dart';
 import 'package:mockito/mockito.dart';
 import '../app_widget.dart';
```

Comment:
absolute imports

Reply:
I've tried using absolute imports in testing but wont work because our package `concordia_navigation` starts at the `lib/` folder while `test/` and `test_driver/` are on the same level so they're not visible under our package name. Not sure if there's another way.

Reply:
> I've tried using absolute imports in testing but wont work because our package `concordia_navigation` starts at the `lib/` folder while `test/` and `test_driver/` are on the same level so they're not visible under our package name. Not sure if there's another way.

Ah ok